RealmSwift/Results.swift

```diff
@@ -26,6 +26,17 @@ import Realm
 // Used for conversion from Objective-C types to Swift types
 private protocol Bridgable  { static func bridging(_ value: Any) -> Self }
 
+// FIXME: Remove once Swift supports `as! Self` casts
+private func forceCastTrampoline<T, U>(_ x: T) -> U {
```

Comment:
Name isn't terribly important here. `forceCastHelper` would be fine. Additional context is given by the fixme above.